exchanges/binance/binance_wrapper.go

```diff
@@ -189,10 +189,12 @@ func (b *Binance) SetDefaults() {
 			},
 		},
 		Subscriptions: subscription.List{
-			{Enabled: true, Channel: subscription.TickerChannel},
-			{Enabled: true, Channel: subscription.AllTradesChannel},
-			{Enabled: true, Channel: subscription.CandlesChannel, Interval: kline.OneMin},
-			{Enabled: true, Channel: subscription.OrderbookChannel, Interval: kline.HundredMilliseconds},
+			{Enabled: true, Asset: asset.Spot, Channel: subscription.TickerChannel, Template: "{{fmt $pair}}@ticker"},
```

Comment:
NOTE 4 me: templating not coming up in config

Reply:
Acknowledging this is a self note, I'm still curious what it means?

Reply:
Oh I was expecting it to have the immutable string show up in the config as well. So that we might be able to customise it on the fly from config loading. But that is not part of this PR or wasn't even suggested as a thing.